cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -140,10 +139,14 @@ class BaseEbsSchema(BaseSchema):
     throughput = fields.Int()
     encrypted = fields.Bool()
 
+
+class RootVolumeSchema(_BaseEbsSchema):
+    """Represent the RootVolume schema."""
+
```

Comment:
Why creating a new class if the class is exactly the same as BaseEbsSchema?

Reply:
The two classes have the same attributes but different methods. I.e. `_BaseEbsSchema` doesn't have the `post_load` method.